pontoon/machinery/openai_service.py

```diff
@@ -20,29 +21,32 @@ def get_translation(
                 f"The target language '{target_language_name}' is not supported."
             )
 
-        informal = (
-            f"You will be provided with text in English, along with its machine-generated translation in {target_language}. "
-            "Your objective is to revise the {target_language} translation to ensure it utilizes simpler language. Adhere to the following guidelines to achieve this:\n"
-            "- Clarity is Key: Ensure the translation conveys the original message in the clearest possible manner, without ambiguity or unnecessary complexity.\n"
-            "- Consistent Simplicity: Maintain a consistent level of simplicity throughout the translation.\n"
-            "The goal is to produce a translation that accurately reflects the original English text, but in a way that is more approachable and easier to understand for all {target_language} speakers."
+        informal = textwrap.dedent(
+            f"""
```

Comment:
Should drop the leading newline.
```suggestion
            f"""\
```